added note for WebView2 replacing WebView (#270)
MicrosoftDocs/WindowsCommunityToolkitDocs: docs/controls/wpf-winforms/WebView.md

# WebView control for Windows Forms and WPF

> Please note that WebView will eventually be replaced by [WebView2] (https://docs.microsoft.com/en-us/microsoft-edge/hosting/webview2) (currently in preview). Thus, this means that WebView has been depreciated within the ToolKit but we are working on bringing the functionality back to WebView2. If you would like to give feedback directly for WebView and help with any open issues, you can do so [here on the Edge repository] (https://github.com/MicrosoftEdge/WebViewFeedback).

Reviewer: Thanks @salmanmkc, we probably want this in a note or warning section to make it clear that it's an auxiliary to the content and stand out a bit more.  Also made a few minor edits to help convey moving forward:
```suggestion
> [!NOTE]
> Please note that WebView will eventually be replaced by [WebView2] (https://docs.microsoft.com/en-us/microsoft-edge/hosting/webview2) (currently in preview). Thus, the WebView has been deprecated within the Toolkit, but we are working on conveying requirements to the WebView2 team. If you would like to give feedback directly for WebView2, you can do so [here on the Edge repository](https://github.com/MicrosoftEdge/WebViewFeedback).
```
